acsys/dpm/__init__.py

```diff
@@ -6,109 +6,90 @@
 import os
 import sys
 import acsys.status
-from acsys.dpm.dpm_protocol import (ServiceDiscovery_request, OpenList_request,
-                                    AddToList_request, RemoveFromList_request,
-                                    StartList_request, StopList_request,
-                                    ClearList_request, RawSetting_struct,
-                                    TextSetting_struct, ScaledSetting_struct,
-                                    ApplySettings_request, Status_reply,
-                                    AnalogAlarm_reply, BasicStatus_reply,
-                                    DigitalAlarm_reply, DeviceInfo_reply,
-                                    Raw_reply, ScalarArray_reply, Scalar_reply,
+from acsys.dpm.dpm_protocol import (ServiceDiscovery_request,
+                                    AddToList_request,
+                                    RemoveFromList_request,
+                                    StartList_request,
+                                    StopList_request,
+                                    ClearList_request,
+                                    RawSetting_struct,
+                                    TextSetting_struct,
+                                    ScaledSetting_struct,
+                                    ApplySettings_request,
+                                    Status_reply, AnalogAlarm_reply,
+                                    BasicStatus_reply,
+                                    DigitalAlarm_reply,
+                                    DeviceInfo_reply, Raw_reply,
+                                    ScalarArray_reply, Scalar_reply,
                                     TextArray_reply, Text_reply,
-                                    ListStatus_reply, ApplySettings_reply,
+                                    ListStatus_reply,
+                                    ApplySettings_reply,
                                     Authenticate_request,
                                     EnableSettings_request,
-                                    TimedScalarArray_reply, Authenticate_reply)
+                                    TimedScalarArray_reply,
+                                    Authenticate_reply,
+                                    unmarshal_reply)
 
-_log = logging.getLogger(__name__)
+_log = logging.getLogger('acsys')
```

Comment:
This, https://docs.python.org/3.6/library/logging.html#logger-objects, recommends using `__name__` for modules to match the package hierarchy. Do you have an idea of why that doesn't work for us?

Reply:
I was being a bit lazy. In my test scripts, I get the `acsys` logger and set the level. I guess the test scripts can grabs each `logger` for each sub package and set the level.

Reply:
Fixed in e3725eb.